src/zip.js

```diff
@@ -110,19 +117,180 @@ function zipJs(functionPath, zipPath) {
   }
   const moduledir = findModuleDir(basedir);
 
+  const requireRegex = /require\(['|"](.*)['|"]\)/gm
```

Comment:
This probably should be `/require\(['|"][^'"]+['|"]\)/gm` otherwise this might fail e.g. if two `require` statements are on the same line, e.g. `require("a"); require("b");`